pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
@php-coder Thanks.
I tried. `filepath.Clean()` can support  Windows-style paths. 
 search for `..` maybe not exact. people may create a dir named `1..2`